optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/PureSingleStatistic.java

```diff
@@ -30,7 +30,7 @@
 import org.optaplanner.benchmark.impl.statistic.single.pickedmovetypestepscore.PickedMoveTypeStepScoreDiffSingleStatistic;
 
 /**
- * 1 statistic of {@link SingleBenchmarkResult}
+ * 1 statistic of {@link SingleBenchmarkResult}.
```

Comment:
All sentences should end with a dot (or question mark etc), but this is not a sentence because it has no verb, so we shouldn't add a dot here